pandas/core/groupby/ops.py

```diff
@@ -299,6 +299,270 @@ def get_result_dtype(self, dtype: DtypeObj) -> DtypeObj:
     def uses_mask(self) -> bool:
         return self.how in self._MASKED_CYTHON_FUNCTIONS
 
+    @final
```

Comment:
can we move WrappedCythonOps the class to a new module, maybe cython_ops.py? (can do it in this PR or pre-cursor)

Reply:
at some point.  naming-wise i actually think it makes sense for it to be in ops.py and everything else to be somewhere else, but im not sure where that would be (the relationship between Grouper vs BaseGrouper still confuses me)